src/pages/workspaces/workspace/analysis/runtime-utils.js

```diff
@@ -506,3 +508,17 @@ export const cloudProviders = {
 export const isGcpContext = ({ cloudProvider }) => cloudProvider === cloudProviders.gcp.label
 export const isAzureContext = ({ cloudProvider }) => cloudProvider === cloudProviders.azure.label
 
+//TODO: fields isAppStatus? LeoLabel? isRuntimeStatus?
+export const runtimeStatuses = {
+  running: { label: 'Running', leoLabel: 'Running', canChangeCompute: true },
```

Comment:
Is `leoLabel` used anywhere? I'm also confused because it seems to be identical to `label`.

Reply:
This is a first pass at making a complex `status` type. `leoLabel` will be different in a few cases once this type is used. For example, `Starting` is what leo labels the status for which the UI wants to display `Resuming` to the user.